recipes/openssl/3.x.x/conanfile.py

```diff
@@ -111,10 +124,21 @@ def configure(self):
         self.settings.rm_safe("compiler.libcxx")
         self.settings.rm_safe("compiler.cppstd")
 
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
     def requirements(self):
         if not self.options.no_zlib:
             self.requires("zlib/[>=1.2.11 <2]")
 
+    def validate(self):
```

Comment:
Better avoid stylistic refactors, like moving a method, otherwise it is more difficult to review the actual functional changes of the PR